### PR TITLE
test: test for coding bounty

### PR DESCRIPTION
**Pull request by okhot**
## Describe your changes #242
Added Test for Coding Bounty
Screenshot 2024-02-16 at 15 16 36

## Issue ticket number and link

- [X] Test (non-breaking change which adds functionality)

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by aliraza556** [comment]
Hi @okhot,
Have you tested this locally to confirm whether Jest has passed for the component? If so, could you please provide a screenshot of the terminal output for reference?

**Comment by aliraza556** [comment]
Your description's output screenshot link does not work when I click he open the PR again. Please provide a new screenshot link.

File: src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx [hunk 1]
```diff
@@ -325,4 +329,110 @@ describe('MobileView component', () => {
...
+    fireEvent.click(checkBox[0]);
+
+    (async () => {
+      await waitFor(() => {
+        expect(screen.getByText('Mark as Paid'));
+      });
+    })();
+  });
+
+  it('Test that if sufficient funds are available, bounty is paid', async () => {
```
Comment (elraphty) [inline src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx:420]: @okhot This does not prove that Payment  action can be taken if the organization has sufficient funds

Comment (okhot) [reply src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx:420]: I've created a bounty and tried the Payout flow and the payment always goes through with or without sufficient funds. That's what got me stuck there.

Comment (elraphty) [reply src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx:420]: > I've created a bounty and tried the Payout flow and the payment always goes through with or without sufficient funds. That's what got me stuck there.

Are you getting this error on the Pay Bounty button, or the Mark As Paid Button?

Comment (okhot) [reply src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx:420]: Screenshot 2024-02-16 at 18 55 53

Comment (elraphty) [reply src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx:420]: > Screenshot 2024-02-16 at 18 55 53

oh! Yeah, the Mark as Paid button will always be displayed,  test that the mark as paid button updates the bounty paid state, and I will update the button test ticket, thanks.

**Pull request merged**